pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -826,6 +847,18 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 				template.Cors = serverSet.Cors
 			}
 
+			if cr.Status.Applied.CommonConfig.StartupStrategy != nil {
+
+				if cr.Status.Applied.CommonConfig.StartupStrategy.StrategyName != "" {
+					template.StartupStrategy.StrategyName = cr.Status.Applied.CommonConfig.StartupStrategy.StrategyName
+				} else {
+					template.StartupStrategy.StrategyName = api.OpenshiftStartupStrategy
+				}
+
+			} else {
+				template.StartupStrategy = &api.StartupStrategy{StrategyName: api.OpenshiftStartupStrategy}
```

Comment:
is it needed as default value is being set here https://github.com/kiegroup/kie-cloud-operator/pull/645/files#diff-29ad51b906bc49dd54db094a3b626e8e4a04628d03562e6c0d45db0e391ac2fbR1384
?

same for the console one.

Reply:
Yes we can remove